.github/workflows/_integration-tests.yml

```
@@ -209,7 +209,13 @@ jobs:
         -   uses: actions/setup-python@v5
             with:
                 python-version: ${{ inputs.python-version }}
-        -   run: echo "HATCH_PYTHON=${{ inputs.python-version }}" >> $GITHUB_ENV
+        -   run: |
+                echo "HATCH_PYTHON=${{ inputs.python-version }}" >> $GITHUB_ENV
+                echo "PIP_ONLY_BINARY=psycopg2-binary" >> $GITHUB_ENV
+        # mac + 3.9 requires postgresql to be installed manually so that the psycopg2-binary wheel can be found
+        -   if: inputs.python-version == '3.9' && inputs.os == 'macos-14'
+            run: |
+                brew install postgresql
```

Minor nit, we don't need two lines for a one-liner.
```suggestion
            run: brew install postgresql
```